packages/babel-parser/test/fixtures/flow/class-properties/declare-field-computed/input.js

```diff
@@ -0,0 +1,3 @@
+class A {
+  declare [foo]
+}
```

Comment:
Flow does not support declared computed class fields yet: https://flow.org/try/#0G4QwTgBAZg9jEF4IEYDcAoAxgGxAZzwgEEIBvdCCAEwFMdwaIBtWGAXXQF8g

Reply:
It's not released yet.

Looking at the source code it doesn't seem to be disallowed (https://github.com/facebook/flow/commit/11b7adbf85ea237cfc01d1b8dc6f6dcdbc299157#diff-ca4a151bb3ce5bd8c11f1bc3b00036ebR619), but I don't see any test to confirm my assumption.

cc @@mroch should they be supported?

Reply:
change the version selector in the top right to `master` and it parses.

It does error with "Computed property keys not supported.", but that's a type error, not a parse error (you can tell the difference on the CLI, but on Try Flow it's only in the JSON tab (`"kind": "infer"`))

Reply:
Ok, thanks :+1: